refresh/config.go

```diff
@@ -24,7 +26,13 @@ type Configuration struct {
 }
 
 func (c *Configuration) FullBuildPath() string {
-	return path.Join(c.BuildPath, c.BinaryName)
+	path := path.Join(c.BuildPath, c.BinaryName)
+	if runtime.GOOS == "windows" {
+		if !strings.HasSuffix(path, ".exe") {
```

Comment:
If you open a pr to handle the case and change the path variable to something else would be great.